catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/FilterReader.java

```diff
@@ -35,14 +50,38 @@ public FilterReader() throws JAXBException {
   @SuppressWarnings("unchecked")
   private <T> JAXBElement<T> unmarshal(InputStream inputStream, Class<T> tClass)
       throws JAXBException {
+    SAXParser parser;
+    try {
+      parser = factory.newSAXParser();
```

Comment:
✏️ You can get rid of the factory field and just call 
`parser = XMLUtils.getInstance().getSecureSAXParser(true);`